### PR TITLE
Link to GitBook docs

### PR DESCRIPTION
**Pull request by taleldayekh**
This PR is part of the "Documentation for the pragmatic developer" issue.

An infobox has been added on the index page and on the "Web Application Tutorial" page which notifies visitors about the current process of updating the docs and provides a link to the new (incomplete) GitBook version.

### CODE REVIEW

File: docs/webui_tutorial.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Web Application Tutorial
 .. toctree::
   :maxdepth: 2
 
+.. NOTE::
+      We are currently reworking our documentation for Raiden and just finished updating this tutorial for the web application. You can view the updated web application tutorial in this `new but incomplete version of the documentation <http://doc.raiden.network/>`_.
```
Comment (palango) [inline docs/webui_tutorial.rst]: I don't see anything on http://doc.raiden.network/

Comment (taleldayekh) [reply docs/webui_tutorial.rst]: Hmm, perhaps the DNS configuration hasn't gone through yet.

Comment (taleldayekh) [reply docs/webui_tutorial.rst]: Fyi, I did one more commit with text mentioning the Wizard and links to GitBook. However I am still waiting for the DNS configuration. I'll re-request a review as soon as that is working.

Comment (taleldayekh) [reply docs/webui_tutorial.rst]: GitBook links are live now and pointing to the correct sections.

Comment (palango) [reply docs/webui_tutorial.rst]: https://docs.raiden.network/ works for me, but not https://doc.raiden.network/

Comment (taleldayekh) [reply docs/webui_tutorial.rst]: I made a new commit a couple of minutes ago changing all links to the https://docs.raiden.network.

Comment (palango) [reply docs/webui_tutorial.rst]: Ah sorry.

**Pull request merged**